src/pages/HomePage/FeaturedProducts/FeaturedProducts.css

```diff
@@ -0,0 +1,90 @@
+.FeaturedProducts {
```

Comment:
1. it must be scss. please change all your css to scss
2. we use kebab-case for classes:
featured-products instead of FeaturedProducts
chair-box instead of chairBox
and so on
please fix it in all scss files